eslint.config.mjs

```diff
@@ -1,38 +1,47 @@
 import guardian from '@guardian/eslint-config';
+import globals from 'globals';
 
 export default [
 	{
 		ignores: [
 			'node_modules',
 			'dist',
+            'server-dist',
 
-			'customize.js',
 			'rollup.config.js',
 			'webpack.*js',
 		],
 	},
 	...guardian.configs.recommended,
 	...guardian.configs.jest,
 	{
+        ignores: ['eslint.config.mjs'],
+        languageOptions: {
```

Comment:
this section heavily inspired by https://github.com/guardian/commercial/blob/main/eslint.config.mjs#L16